keycloak/keycloak-admin-ui
# Realm settings(login): adds login tab and tests

## Motivation

JIRA: https://issues.redhat.com/browse/APPDUX-920
Github: Closes #530 

## Brief Description

Adds login tab to realm settings.

## Verification Steps


1. Go to Realm Settings.
2. Click on Login tab.
3. Toggle switches and verify that save functionality works as expected.

## Checklist:


## Additional Notes
